defaults/main.yml

```diff
@@ -146,9 +195,9 @@ neofs_ir__config: []
 neofs_ir__group_config: []
 neofs_ir__host_config: []
 
-neofs_ir__combined_config: '{{ 
+neofs_ir__combined_config: '{{
   neofs_ir__default_config +
-  neofs_ir__config + 
+  neofs_ir__config +
```

Comment:
Whitespace change -> separate commit.

Reply:
I cannot do it due to the vim settings. It automatically cuts off whitespaces at the end of a line.

Reply:
You can. You really really can. Irrespective of `$EDITOR` settings there are at least seven ways to deal with this problem. And should be solved.

Reply:
Done